.changeset/dirty-crabs-try.md

```diff
@@ -5,6 +5,6 @@
 "section": deprecation
 ---
 
-Deprecated SummarizerStopReason, ISummarizeEventProps, and ISummarizerEvents
+SummarizerStopReason, ISummarizeEventProps, and ISummarizerEvents are now deprecated
 
-`SummarizerStopReason`, `ISummarizeEventProps`, and `ISummarizerEvents` have all been deprecated from the `"@fluidframework/container-runtime"` package. Please migrate all uses of these APIs to their respective copies in the `"@fluidframework/container-runtime-definitions"` package.
+`SummarizerStopReason`, `ISummarizeEventProps`, and `ISummarizerEvents` have all been deprecated from the `"@fluidframework/container-runtime"` package. Please migrate all uses of these APIs to their respective implementations in the `"@fluidframework/container-runtime-definitions"` package.
```

Comment:
They're all types with no runtime effect, so accounting for the distinction we usually do between the "definitions" and "implementations" of things in separate packages, I think this can help avoid potential confusion.
```suggestion
`SummarizerStopReason`, `ISummarizeEventProps`, and `ISummarizerEvents` have all been deprecated from the `"@fluidframework/container-runtime"` package. Please migrate all uses of these APIs to their counterparts in the `"@fluidframework/container-runtime-definitions"` package.
```